### PR TITLE
update download links to match the new style expected by the stacks

### PR DESCRIPTION
**Pull request by peetucket**
anyone know why the three tests are failing?  i see them failing on other branches too, is there a known issue?

### CODE REVIEW

**Comment by mejackreed** [comment]
Various reasons, but mostly having to do with last minute changes to the implemented IIIF-Auth.

**Comment by peetucket** [comment]
so does that mean no changes (even simple ones like this) can be made before more time is devoted to this project?

**Comment by mejackreed** [comment]
Yes. I need a few hours allocated to working on this.

File: app/assets/javascripts/modules/download_panel.js [hunk 1]
```diff
@@ -22,14 +22,14 @@
         .append('<a class="download-link" href="' + imageData['@id'] +
         '/full/!' + disableDownloadWidthCutoff + ',' +
         disableDownloadWidthCutoff +
-        '/0/default.jpg" download>Download Thumbnail</a>'));
+        '/0/default.jpg?download=true" download>Download Thumbnail</a>'));
```
Comment (jkeck) [inline app/assets/javascripts/modules/download_panel.js]: I noticed that we missed this one.  I assume we want the `download=true` for thumbnails as well as the other downloadable derivatives?

Comment (mejackreed) [inline app/assets/javascripts/modules/download_panel.js]: Yep seems that way. @peetucket ?

Comment (peetucket) [inline app/assets/javascripts/modules/download_panel.js]: "no thumb for you!"
sorry, yes, I am sure this was just missed

**Pull request merged**